tsdb/head.go

```diff
@@ -1749,6 +1764,11 @@ func (s *stripeSeries) getByHash(hash uint64, lset labels.Labels) *memSeries {
 }
 
 func (s *stripeSeries) getOrSet(hash uint64, lset labels.Labels, createSeries func() *memSeries) (*memSeries, bool, error) {
+	// Intern all labels
+	if s.interner != nil {
+		s.interner.Intern(lset)
+	}
```

Comment:
Interning happens before the `PreCreation` hook here, so if that fails we would be leaking those series, right?